packages/bitcore-node/src/models/webhook.ts

```diff
@@ -0,0 +1,54 @@
+import { ObjectID } from 'mongodb';
```

Comment:
ObjectID is deprecated in the latest MongoDB versions and seldom used in 4.0>, consider using ObjectId